app/controllers/catalog_controller.rb

```diff
@@ -321,8 +321,7 @@ def collection_search
 
   def metadata_only_collections(params)
     meta_colls = []
-    solr_params = {}
-    solr_params[:fq] = ["{!join from=collections_tesim to=id}#{metadata_only_fquery}"]
+    solr_params = { q: "{!join from=collections_tesim to=id}#{metadata_only_fquery}", fq: 'type_tesim:Collection', rows: 300 }
```

Comment:
Is there any side effects when merging the parameter :q with that :q param passing in hash params in the line#325? Maybe we can try to use something like 
`{ fq: ["{!join from=collections_tesim to=id}#{metadata_only_fquery}", 'type_tesim:Collection'], rows: 300 }`?

Reply:
@lsitu - I don't know if there's any side effect.  I could try your suggestion and deploy to staging to test.  Last time, it has problem on staging with the old fq.

Reply:
@mcritchlow I'll be off line tomorrow and I don't want to block the PR. Could you give it a pass if you don't see any issues with the query? I am not sure how merging of two :q parameter works in https://github.com/ucsdlib/damspas/pull/503/files#diff-956e46b33fb5307990c4e1a4b5fd86ccR325.

Reply:
@lsitu - I reverted the change to my query and deployed to staging. The search results worked fine. Both collection view and search view access label matched. It’s hard to have a test for it since staging will return larger results then local test with only few records created. There are already some tests to catalog spec and dams-collection test for search result access label. Those tests passed for both of our queries. They just behaved differently on staging.